### PR TITLE
add lottie library to readme

### PR DESCRIPTION
**Pull request by maracuja-juice**
I saw in your `build.gradle` file that you use the Lottie library so I added it to the Readme.

Would be interested to know though where exactly you use the library. I mean I can't find that much animations.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -104,6 +104,7 @@ _Ads currently not available._
 - [**Toasty**](https://github.com/GrenderG/Toasty) for displaying error/success messages
 - [**ShapedImageView**](https://github.com/gavinliu/ShapedImageView) for round avatars
 - [**Material-About-Library**](https://github.com/daniel-stoneuk/material-about-library) for the about screen
+- [**Lottie**](https://github.com/airbnb/lottie-android) for animations
```
Comment (k0shk0sh) [inline README.md]: please append it to last.

**Comment by k0shk0sh** [comment]
its only used in promo code.

**Comment by maracuja-juice** [comment]
Okay. Like this?

Also why is `Fabric` listed under Open-Source Libraries? (I thought that that's not open source)

**Comment by k0shk0sh** [comment]
that's a good point, I remember back then it was libs used can't remember when we changed that :D. you could exclude it.

**Pull request merged**